pkg/requestgen/generator.go

```diff
@@ -140,3 +176,31 @@ func keyUsagesFromAttributes(usagesCSV string) []cmapi.KeyUsage {
 
 	return keyUsages
 }
+
+// expand executes os.Expand on the given csv with volume context variables
+// provided by the metadata.
+func expand(meta metadata.Metadata, csv string) (string, error) {
+	vars := map[string]string{
+		"PodName":      meta.VolumeContext[csiapi.K8sVolumeContextKeyPodName],
+		"PodNamespace": meta.VolumeContext[csiapi.K8sVolumeContextKeyPodNamespace],
+		"PodUID":       meta.VolumeContext[csiapi.K8sVolumeContextKeyPodUID],
```

Comment:
Would it be more in-line with environment variable styling to use `POD_NAME`, `POD_NAMESPACE` etc? I don't have a strong opinion, but I found it didn't look quite right having lowercase and no underscores. 

These are documented anyway so feel free to go with these if you prefer.

Reply:
Yes, I agree. Let's change them to include underscores and use upper case in the documentation.

Reply:
Done :slightly_smiling_face: